models/class.metrilo-order-data.php

```diff
@@ -16,11 +16,18 @@ public function get_orders($chunk_id)
     {
         $orders = [];
         $order_ids = $this->db_connection->get_col(
-            "SELECT id FROM {$this->db_connection->posts}
-        WHERE post_type = 'shop_order'
-        ORDER BY id ASC
-        limit {$this->chunk_items}
-        offset {$chunk_id}"
+            $this->db_connection->prepare(
+        "
+                SELECT id
+                FROM $this->db_connection->posts
+                WHERE post_type = 'shop_order'
+                ORDER BY id ASC
+                limit '%d'
+                offset '%d'
```

Comment:
`limit` and `offset` should be integers. No need to wrap them in quotes in the SQL.

Reply:
Fixed.